app/src/docs/v1.api-spec.yaml

```diff
@@ -50,6 +1095,663 @@ components:
+      type: object
+      description: Form definition updated
+      properties:
+        formId:
+          type: string
+          example: d6944dd9-1c0c-4915-971d-542dda14eb7f
+        description:
+          type: string
+          example: This is an attestation form for Agriculture and Seafood
+        createdBy:
+          type: string
+          example: migration-011
+        createdAt:
+          type: string
+          example: 2020-06-04T18:49:20.672Z
+        updatedBy:
+          type: string
+          example: migration-012
+        updatedAt:
+          type: string
+          example: 2020-06-04T18:49:20.672Z
+        metadata:
+          type: object
+          $ref: '#/components/schemas/Form'
+        versions:
+          type: array
+          items:
+            type: object
+            $ref: '#/components/schemas/FormChanges'
+
+    FormChanges:
+      type: object
+      properties:
+        formVersionId:
+          type: string
+          example: 1
+        formId:
+          type: string
+          example: d6944dd9-1c0c-4915-971d-542dda14eb7f
+        changes:
+          type: string
+          example: 'this is a new version'
+        statusCodes:
+            type: array
+            items:
+              type: object
+              $ref: '#/components/schemas/StatusCode'
+
+
+
+    StatusCode:
+      type: object
+      properties:
+        code:
+          type: string
+          example: SUBMITTED
+        display:
+          type: string
+          example: Submitted
+        enabled:
+          type: boolean
+          example: true
+        nextCodes:
+          type: array
+          items:
+            type: string
+          example: ['ASSIGNED', 'COMPLETED']
+        allowedClassifications:
+          type: array
+          items:
+            type: string
+          example: []
+        createdBy:
+          type: string
+          example: migration-011
+        createdAt:
+          type: string
+          example: 2020-06-04T18:49:20.672Z
+        updatedBy:
+          type: string
+          example: null
+        updatedAt:
+          type: string
+          example: 2020-06-04T18:49:20.672Z
+
+
+    StatusPost:
+      type: object
+      properties:
+        code:
+          type: string
+          example: ASSIGNED
+        assignedTo:
+          type: string
+          example: John Smith
+        assignedToEmail:
+          type: string
+          example: jsmith@aol.com
+        actionDate:
+          type: string
+          format: date
+          example: 2020-06-01
+        notes:
+          type: array
+          items:
+            type: object
+            properties:
+              note:
+                type: string
+                example: 'This Client is ready for a phone call'
+
+
+    NotePost:
+            type: object
+            properties:
+              note:
+                type: string
+                example: 'This Client is ready for a phone call'
+
+
```

Comment:
Let's make sure to run some kind of indentation checker before this gets merged in. Spacing is very important in YAML files and this can easily break the entire spec if not dealt with. VSCode and the editor.swagger.io tool should be able to ensure indentation integrity.